BAZProjectC1/Components/BasicInfoMovie/BasicInfoMovie.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  BasicInfoMovie.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 23/09/22.
+//
+
+import UIKit
+
+class BasicInfoMovie: UIView {
```

Comment:
El nombre de esta clase da a entender que sirve para presentar información, pero solo se está inyectando un color.
¿Es la intención presentar información? ¿O podíamos renombrar la clase de forma que represente lo que hace?, por ejemplo:

```suggestion
class MovieBackgroundView: UIView {
```